### PR TITLE
fix(RHINENG-1485): Move status label from expandable section to issue title

### PR DESCRIPTION
**Pull request by mhuth**
Before PR:

Screenshot from 2024-09-11 16-56-23

Screenshot from 2024-09-11 16-57-11

After PR:

Screenshot from 2024-09-11 16-55-03

Screenshot from 2024-09-11 16-56-03

### CODE REVIEW

File: src/SmartComponents/CompletedTaskDetails/JobResultsDetails/EntryRow.js [hunk 1]
```diff
@@ -1,19 +1,23 @@
...
 import severityMap from '../TaskEntries';
-import { Icon } from '@patternfly/react-core';
+import EntryRowLabel from './EntryRowLabel';
 
 const EntryRow = ({ severity, taskConstantMapper, title }) => {
   const mappedSeverity =
     severityMap[taskConstantMapper][severity.toLowerCase()];
-  const renderIcon = () => {
+
+  const getLabelType = () => {
```
Comment (mkholjuraev) [inline src/SmartComponents/CompletedTaskDetails/JobResultsDetails/EntryRow.js:10]: Probably, this should be converted to a React component and be given proper component name.

File: src/SmartComponents/CompletedTaskDetails/JobResultsDetails/EntryRowLabel.js [hunk 2]
```diff
@@ -7,7 +7,7 @@ const EntryRowLabel = ({ color, icon, text }) => {
     <Label
       color={color}
       icon={icon}
-      style={{ marginTop: '16px', marginBottom: '4px' }}
+      style={{ marginTop: '0.25rem', marginBottom: '0.25rem' }}
```
Comment (mkholjuraev) [inline src/SmartComponents/CompletedTaskDetails/JobResultsDetails/EntryRowLabel.js:10]: Even better approach would be to use PF variables or classes to position the element if possible. More info: https://www.patternfly.org/design-foundations/spacers

File: src/SmartComponents/CompletedTaskDetails/JobResultsDetails/EntryRow.js (hunk 1 above)
Comment (mhuth) [reply src/SmartComponents/CompletedTaskDetails/JobResultsDetails/EntryRow.js]: Done!

File: src/SmartComponents/CompletedTaskDetails/JobResultsDetails/EntryRowLabel.js (hunk 2 above)
Comment (mhuth) [reply src/SmartComponents/CompletedTaskDetails/JobResultsDetails/EntryRowLabel.js]: Done and thanks!  I didn't know about them before now.

**Review by mkholjuraev — APPROVED** [review]
LGTM!

**Pull request merged**